src/__tests__/shared/components/input/TextFieldWithLabelFormGroup.test.tsx

```diff
@@ -18,101 +19,89 @@ describe('text field with label form group', () => {
-      const expectedName = 'test'
-      const expectedRequired = true
-      const wrapper = shallow(
+      render(
         <TextFieldWithLabelFormGroup
-          name={expectedName}
+          name="test"
           label="test"
           value=""
           isEditable
-          isRequired={expectedRequired}
+          isRequired
           onChange={jest.fn()}
         />,
       )
 
-      const label = wrapper.find(Label)
-      expect(label).toHaveLength(1)
-      expect(label.prop('isRequired')).toBeTruthy()
+      expect(screen.getByTitle(/required/i)).toBeInTheDocument()
     })
 
     it('should render a text field', () => {
-      const expectedName = 'test'
-      const wrapper = shallow(
+      render(
         <TextFieldWithLabelFormGroup
-          name={expectedName}
+          name="test"
           label="test"
           value=""
           isEditable
           onChange={jest.fn()}
         />,
       )
 
-      const input = wrapper.find(TextField)
-      expect(input).toHaveLength(1)
+      expect(screen.getByRole('textbox')).toBeInTheDocument()
     })
 
     it('should render disabled is isDisable disabled is true', () => {
-      const expectedName = 'test'
-      const wrapper = shallow(
+      render(
         <TextFieldWithLabelFormGroup
-          name={expectedName}
+          name="test"
           label="test"
           value=""
           isEditable={false}
           onChange={jest.fn()}
         />,
       )
 
-      const input = wrapper.find(TextField)
-      expect(input).toHaveLength(1)
-      expect(input.prop('disabled')).toBeTruthy()
+      expect(screen.getByRole('textbox')).toBeDisabled()
     })
 
     it('should render the proper value', () => {
-      const expectedName = 'test'
       const expectedValue = 'expected value'
-      const wrapper = shallow(
+
+      render(
         <TextFieldWithLabelFormGroup
-          name={expectedName}
+          name="test"
           label="test"
           value={expectedValue}
-          isEditable={false}
+          isEditable
           onChange={jest.fn()}
         />,
       )
 
-      const input = wrapper.find(TextField)
-      expect(input).toHaveLength(1)
-      expect(input.prop('value')).toEqual(expectedValue)
+      expect(screen.getByRole('textbox')).toHaveValue(expectedValue)
     })
   })
 
   describe('change handler', () => {
     it('should call the change handler on change', () => {
-      const expectedName = 'test'
       const expectedValue = 'expected value'
-      const handler = jest.fn()
-      const wrapper = shallow(
+      let callBackValue = ''
+
+      render(
         <TextFieldWithLabelFormGroup
-          name={expectedName}
+          name="test"
           label="test"
-          value={expectedValue}
-          isEditable={false}
-          onChange={handler}
+          value=""
+          isEditable
+          onChange={(e) => {
+            callBackValue += e.target.value
```

Comment:
I'm not sure what was going on here, but the change event was only coming through with the last typed character, i.e. the `e.target.value`s were

1. `e`
2. `x`
3. `p`
4. ...

I don't _think_ that would happen in a real browser environment, right?